### ISSUE TITLE
Not able to sign in for the first time

### ISSUE THREAD

**Issue by vadi2** [photoprism/photoprism#1610]
My first experience using photoprism, I have set a simple password in the docker-compose settings (`123` just to test) but it is not working. Moreover, every time I try to sign in, regardless of container restarts, the time to verify credentials goes up by 5s:

```
photoprism_1  | time="2021-10-08T09:59:36Z" level=debug msg="http: POST /api/v1/session (400) [15.856325488s]"
photoprism_1  | time="2021-10-08T10:00:12Z" level=debug msg="http: POST /api/v1/session (400) [20.861944956s]"
photoprism_1  | time="2021-10-08T10:00:44Z" level=debug msg="http: POST /api/v1/session (400) [25.866296127s]"
```

**Comment by vadi2** [comment]
Changing it to a public server worked, though.

**Comment by graciousgrey** [comment]
This was solved by using admin as username instead of root, right?

**Comment by vadi2** [comment]
It was not, `admin` did not work either.

**Comment by lastzero** [comment]
Note that It's not possible to change the initial password via `PHOTOPRISM_ADMIN_PASSWORD` after PhotoPrism has been started for the first time. You may run `docker-compose exec photoprism photoprism reset` in a terminal to reset your database for a clean start. We also added this information to our docs today. If it is still not working, it may be an issue with a browser plugin, ad blocker, or proxy server in between your browser and PhotoPrism.

**Comment by lastzero** [comment]
Also note that the behaviour of the authentication API is an important security feature to prevent brute force attacks. I'll close this issue since everyone else (including us) is able to sign in. You can open a discussion if you need more help. It doesn't seem to be a "fixable" bug on our side (GitHub issues are for bugs and feature requests only)

**Comment by vadi2** [comment]
```
✦ ❯ docker-compose exec photoprism photoprism reset
ERROR: No container found for photoprism_1

SSDer/Programs/photoprism 
✦ ❯ docker ps
CONTAINER ID   IMAGE          COMMAND                  CREATED        STATUS              PORTS      NAMES
99982751b0be   mariadb:10.5   "docker-entrypoint.s…"   25 hours ago   Up About a minute   3306/tcp   photoprism_mariadb_1

SSDer/Programs/photoprism 
✦ ❯ docker container stop 99982751b0be
99982751b0be

SSDer/Programs/photoprism 
✦ ❯ docker container rm 99982751b0be
99982751b0be

SSDer/Programs/photoprism 
✦ ❯ docker-compose up photoprism
Creating photoprism_mariadb_1 ... done
Recreating photoprism_photoprism_1 ... done
Attaching to photoprism_photoprism_1
photoprism_1  | started as root
photoprism_1  | running as root
photoprism_1  | photoprism start
photoprism_1  | time="2021-10-09T11:44:00Z" level=debug msg="config: running on “AMD Ryzen 5 3600 6-Core Processor”, 34 GB memory detected"
photoprism_1  | time="2021-10-09T11:44:00Z" level=debug msg="config: settings loaded from /photoprism/storage/config/settings.yml "
photoprism_1  | time="2021-10-09T11:44:01Z" level=info msg="http: enabling gzip compression"
photoprism_1  | [GIN-debug] [WARNING] Running in "debug" mode. Switch to "release" mode in production.
photoprism_1  |  - using env:	export GIN_MODE=release
photoprism_1  |  - using code:	gin.SetMode(gin.ReleaseMode)
photoprism_1  | 
photoprism_1  | time="2021-10-09T11:44:01Z" level=debug msg="restore: album yaml files disabled"
photoprism_1  | [GIN-debug] Loaded HTML Templates (8): 
photoprism_1  | 	- app.tmpl
photoprism_1  | 	- favicons.tmpl
photoprism_1  | 	- index.tmpl
photoprism_1  | 	- manifest.json
photoprism_1  | 	- minimal.tmpl
photoprism_1  | 	- rainbow.tmpl
photoprism_1  | 	- share.tmpl
photoprism_1  | 	- 
photoprism_1  | 
photoprism_1  | [GIN-debug] GET    /static/*filepath         --> github.com/gin-gonic/gin.(*RouterGroup).createStaticHandler.func1 (4 handlers)
photoprism_1  | [GIN-debug] HEAD   /static/*filepath         --> github.com/gin-gonic/gin.(*RouterGroup).createStaticHandler.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /favicon.ico              --> github.com/gin-gonic/gin.(*RouterGroup).StaticFile.func1 (4 handlers)
photoprism_1  | [GIN-debug] HEAD   /favicon.ico              --> github.com/gin-gonic/gin.(*RouterGroup).StaticFile.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /manifest.json            --> github.com/photoprism/photoprism/internal/server.registerRoutes.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /sw.js                    --> github.com/photoprism/photoprism/internal/server.registerRoutes.func2 (4 handlers)
photoprism_1  | [GIN-debug] GET    /rainbow                  --> github.com/photoprism/photoprism/internal/server.registerRoutes.func3 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/config            --> github.com/photoprism/photoprism/internal/api.GetConfig.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/config/options    --> github.com/photoprism/photoprism/internal/api.GetConfigOptions.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/config/options    --> github.com/photoprism/photoprism/internal/api.SaveConfigOptions.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/settings          --> github.com/photoprism/photoprism/internal/api.GetSettings.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/settings          --> github.com/photoprism/photoprism/internal/api.SaveSettings.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/users/:uid/password --> github.com/photoprism/photoprism/internal/api.ChangePassword.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/session           --> github.com/photoprism/photoprism/internal/api.CreateSession.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/session/:id       --> github.com/photoprism/photoprism/internal/api.DeleteSession.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/accounts          --> github.com/photoprism/photoprism/internal/api.SearchAccounts.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/accounts/:id      --> github.com/photoprism/photoprism/internal/api.GetAccount.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/accounts/:id/folders --> github.com/photoprism/photoprism/internal/api.GetAccountFolders.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/accounts/:id/share --> github.com/photoprism/photoprism/internal/api.ShareWithAccount.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/accounts          --> github.com/photoprism/photoprism/internal/api.CreateAccount.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/accounts/:id      --> github.com/photoprism/photoprism/internal/api.DeleteAccount.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/accounts/:id      --> github.com/photoprism/photoprism/internal/api.UpdateAccount.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/t/:thumb/:token/:size --> github.com/photoprism/photoprism/internal/api.GetThumb.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/dl/:hash          --> github.com/photoprism/photoprism/internal/api.GetDownload.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/videos/:hash/:token/:type --> github.com/photoprism/photoprism/internal/api.GetVideo.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/zip               --> github.com/photoprism/photoprism/internal/api.CreateZip.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/zip/:filename     --> github.com/photoprism/photoprism/internal/api.DownloadZip.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/photos            --> github.com/photoprism/photoprism/internal/api.SearchPhotos.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/geo               --> github.com/photoprism/photoprism/internal/api.SearchPhotosGeo.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/photos/:uid       --> github.com/photoprism/photoprism/internal/api.GetPhoto.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/photos/:uid/yaml  --> github.com/photoprism/photoprism/internal/api.GetPhotoYaml.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/photos/:uid       --> github.com/photoprism/photoprism/internal/api.UpdatePhoto.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/photos/:uid/dl    --> github.com/photoprism/photoprism/internal/api.GetPhotoDownload.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/photos/:uid/links --> github.com/photoprism/photoprism/internal/api.GetPhotoLinks.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/photos/:uid/links --> github.com/photoprism/photoprism/internal/api.CreatePhotoLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/photos/:uid/links/:link --> github.com/photoprism/photoprism/internal/api.UpdatePhotoLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/photos/:uid/links/:link --> github.com/photoprism/photoprism/internal/api.DeletePhotoLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/photos/:uid/approve --> github.com/photoprism/photoprism/internal/api.ApprovePhoto.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/photos/:uid/like  --> github.com/photoprism/photoprism/internal/api.LikePhoto.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/photos/:uid/like  --> github.com/photoprism/photoprism/internal/api.DislikePhoto.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/photos/:uid/label --> github.com/photoprism/photoprism/internal/api.AddPhotoLabel.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/photos/:uid/label/:id --> github.com/photoprism/photoprism/internal/api.RemovePhotoLabel.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/photos/:uid/label/:id --> github.com/photoprism/photoprism/internal/api.UpdatePhotoLabel.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/moments/time      --> github.com/photoprism/photoprism/internal/api.GetMomentsTime.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/files/:hash       --> github.com/photoprism/photoprism/internal/api.GetFile.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/photos/:uid/files/:file_uid --> github.com/photoprism/photoprism/internal/api.DeleteFile.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/markers/:marker_uid --> github.com/photoprism/photoprism/internal/api.UpdateMarker.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/markers/:marker_uid/subject --> github.com/photoprism/photoprism/internal/api.ClearMarkerSubject.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/photos/:uid/files/:file_uid/primary --> github.com/photoprism/photoprism/internal/api.PhotoPrimary.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/photos/:uid/files/:file_uid/unstack --> github.com/photoprism/photoprism/internal/api.PhotoUnstack.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/albums            --> github.com/photoprism/photoprism/internal/api.SearchAlbums.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/albums/:uid       --> github.com/photoprism/photoprism/internal/api.GetAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/albums/:uid/t/:token/:size --> github.com/photoprism/photoprism/internal/api.AlbumCover.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/albums            --> github.com/photoprism/photoprism/internal/api.CreateAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/albums/:uid       --> github.com/photoprism/photoprism/internal/api.UpdateAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/albums/:uid       --> github.com/photoprism/photoprism/internal/api.DeleteAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/albums/:uid/dl    --> github.com/photoprism/photoprism/internal/api.DownloadAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/albums/:uid/links --> github.com/photoprism/photoprism/internal/api.GetAlbumLinks.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/albums/:uid/links --> github.com/photoprism/photoprism/internal/api.CreateAlbumLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/albums/:uid/links/:link --> github.com/photoprism/photoprism/internal/api.UpdateAlbumLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/albums/:uid/links/:link --> github.com/photoprism/photoprism/internal/api.DeleteAlbumLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/albums/:uid/like  --> github.com/photoprism/photoprism/internal/api.LikeAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/albums/:uid/like  --> github.com/photoprism/photoprism/internal/api.DislikeAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/albums/:uid/clone --> github.com/photoprism/photoprism/internal/api.CloneAlbums.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/albums/:uid/photos --> github.com/photoprism/photoprism/internal/api.AddPhotosToAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/albums/:uid/photos --> github.com/photoprism/photoprism/internal/api.RemovePhotosFromAlbum.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/labels            --> github.com/photoprism/photoprism/internal/api.SearchLabels.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/labels/:uid/t/:token/:size --> github.com/photoprism/photoprism/internal/api.LabelCover.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/labels/:uid       --> github.com/photoprism/photoprism/internal/api.UpdateLabel.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/labels/:uid/links --> github.com/photoprism/photoprism/internal/api.GetLabelLinks.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/labels/:uid/links --> github.com/photoprism/photoprism/internal/api.CreateLabelLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/labels/:uid/links/:link --> github.com/photoprism/photoprism/internal/api.UpdateLabelLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/labels/:uid/links/:link --> github.com/photoprism/photoprism/internal/api.DeleteLabelLink.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/labels/:uid/like  --> github.com/photoprism/photoprism/internal/api.LikeLabel.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/labels/:uid/like  --> github.com/photoprism/photoprism/internal/api.DislikeLabel.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/folders/t/:uid/:token/:size --> github.com/photoprism/photoprism/internal/api.FolderCover.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/folders/originals --> github.com/photoprism/photoprism/internal/api.GetFolders.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/folders/originals/*path --> github.com/photoprism/photoprism/internal/api.GetFolders.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/folders/import    --> github.com/photoprism/photoprism/internal/api.GetFolders.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/folders/import/*path --> github.com/photoprism/photoprism/internal/api.GetFolders.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/subjects          --> github.com/photoprism/photoprism/internal/api.SearchSubjects.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/subjects/:uid     --> github.com/photoprism/photoprism/internal/api.GetSubject.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/subjects/:uid     --> github.com/photoprism/photoprism/internal/api.UpdateSubject.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/subjects/:uid/like --> github.com/photoprism/photoprism/internal/api.LikeSubject.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/subjects/:uid/like --> github.com/photoprism/photoprism/internal/api.DislikeSubject.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/faces             --> github.com/photoprism/photoprism/internal/api.SearchFaces.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/faces/:id         --> github.com/photoprism/photoprism/internal/api.GetFace.func1 (4 handlers)
photoprism_1  | [GIN-debug] PUT    /api/v1/faces/:id         --> github.com/photoprism/photoprism/internal/api.UpdateFace.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/upload/:path      --> github.com/photoprism/photoprism/internal/api.Upload.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/import/*path      --> github.com/photoprism/photoprism/internal/api.StartImport.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/import            --> github.com/photoprism/photoprism/internal/api.CancelImport.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/index             --> github.com/photoprism/photoprism/internal/api.StartIndexing.func1 (4 handlers)
photoprism_1  | [GIN-debug] DELETE /api/v1/index             --> github.com/photoprism/photoprism/internal/api.CancelIndexing.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/photos/approve --> github.com/photoprism/photoprism/internal/api.BatchPhotosApprove.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/photos/archive --> github.com/photoprism/photoprism/internal/api.BatchPhotosArchive.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/photos/restore --> github.com/photoprism/photoprism/internal/api.BatchPhotosRestore.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/photos/private --> github.com/photoprism/photoprism/internal/api.BatchPhotosPrivate.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/photos/delete --> github.com/photoprism/photoprism/internal/api.BatchPhotosDelete.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/albums/delete --> github.com/photoprism/photoprism/internal/api.BatchAlbumsDelete.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/batch/labels/delete --> github.com/photoprism/photoprism/internal/api.BatchLabelsDelete.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/photo         --> github.com/photoprism/photoprism/internal/api.GetSvg.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/raw           --> github.com/photoprism/photoprism/internal/api.GetSvg.func2 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/file          --> github.com/photoprism/photoprism/internal/api.GetSvg.func3 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/video         --> github.com/photoprism/photoprism/internal/api.GetSvg.func4 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/label         --> github.com/photoprism/photoprism/internal/api.GetSvg.func5 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/portrait      --> github.com/photoprism/photoprism/internal/api.GetSvg.func6 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/folder        --> github.com/photoprism/photoprism/internal/api.GetSvg.func7 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/album         --> github.com/photoprism/photoprism/internal/api.GetSvg.func8 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/broken        --> github.com/photoprism/photoprism/internal/api.GetSvg.func9 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/svg/uncached      --> github.com/photoprism/photoprism/internal/api.GetSvg.func10 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/status            --> github.com/photoprism/photoprism/internal/api.GetStatus.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/errors            --> github.com/photoprism/photoprism/internal/api.GetErrors.func1 (4 handlers)
photoprism_1  | [GIN-debug] POST   /api/v1/feedback          --> github.com/photoprism/photoprism/internal/api.SendFeedback.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /api/v1/ws                --> github.com/photoprism/photoprism/internal/api.Websocket.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /s/:token                 --> github.com/photoprism/photoprism/internal/api.Shares.func1 (4 handlers)
photoprism_1  | [GIN-debug] GET    /s/:token/:share          --> github.com/photoprism/photoprism/internal/api.Shares.func2 (4 handlers)
photoprism_1  | [GIN-debug] GET    /s/:token/:share/preview  --> github.com/photoprism/photoprism/internal/api.SharePreview.func1 (4 handlers)
photoprism_1  | [GIN-debug] HEAD   /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] GET    /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PUT    /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] POST   /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PATCH  /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] DELETE /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] OPTIONS /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] MKCOL  /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] COPY   /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] MOVE   /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] LOCK   /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] UNLOCK /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PROPFIND /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PROPPATCH /originals/*path          --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | time="2021-10-09T11:44:01Z" level=info msg="webdav: /originals/ enabled, waiting for requests"
photoprism_1  | [GIN-debug] HEAD   /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] GET    /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PUT    /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] POST   /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PATCH  /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] DELETE /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] OPTIONS /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] MKCOL  /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] COPY   /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] MOVE   /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] LOCK   /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] UNLOCK /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PROPFIND /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | [GIN-debug] PROPPATCH /import/*path             --> github.com/photoprism/photoprism/internal/server.WebDAV.func2 (5 handlers)
photoprism_1  | time="2021-10-09T11:44:01Z" level=info msg="webdav: /import/ enabled, waiting for requests"
photoprism_1  | time="2021-10-09T11:44:01Z" level=info msg="http: starting web server at 0.0.0.0:2342"
photoprism_1  | time="2021-10-09T11:44:02Z" level=debug msg="session: open /photoprism/storage/cache/sessions.json: no such file or directory"
photoprism_1  | time="2021-10-09T11:44:58Z" level=debug msg="http: POST /api/v1/session (400) [55.850209682s]"

```

I have a suspicion that your authentication API is misbehaving, but since you are so keen on closing the issue, I'm not going to spend any energy to fight this. I'll just say that it's a rather unpleasant introduction to the project that even a simple login doesn't work right off the bat.

**Comment by lastzero** [comment]
We are happy to provide personal installation support to our sponsors. I've closed the issue because it doesn't seems to be a bug, at least I don't know what we should change based on your report. It seems to be a local installation issue.

Note we've almost instantly updated our docs for you. You now also get a quick response on a weekend. I'm keen on making you a happy user instead of leaving my desk to enjoy the sun.

It seems strange only mariadb was running and not photoprism. Any reason the operating system could have killed the container? Memory issues? File permissions?

When the photoprism container is not running, you can spawn a new instance by running the command like this (see Docker documentation):

```
docker-compose run photoprism photoprism reset
```

**Comment by lastzero** [comment]
Do you still need help? It's Sunday evening and I'm still waiting here to help you.

**Comment by vadi2** [comment]
I'm good. I don't believe the approach taken here is a particularly healthy open-source model. This seems stressful for you and it's not enticing for me to engage in it.

**Comment by lastzero** [comment]
What do you expect? Your comment is totally off the charts when considering that we provide this for free, you ask for my time and everyone else doesn't have ANY issues:

>  I'm not going to spend any energy to fight this. I'll just say that it's a rather unpleasant introduction to the project that even a simple login doesn't work right off the bat.

Cheers.

**Comment by rickysarraf** [comment]
>
> ✦ ❯ docker-compose exec photoprism photoprism reset
> ERROR: No container found for photoprism_1
> 


Not sure what is causing the problem at your end. But I think you'd want to use something like: `docker exec -it photoprism_photoprism_1 /bin/bash`

These specific names have been working fine for me from the very beginning and I guess it applies to the many of other PhotoPrism Users too.

```
rrs@lenovo:~/photoprism-repo (stable-211010-986b0f7+kvalev)$ usystemctl cat photoprism-cleanup
# /home/rrs/.config/systemd/user/photoprism-cleanup.service
[Unit]
Description=Trigger PhotoPrism cleanup

[Service]
Type=simple
ExecStartPre=/bin/bash -xc '/bin/systemctl --user is-active --quiet photoprism-index.service && exit 1 || exit 0'
ExecStartPre=/bin/bash -xc '/bin/systemctl --user is-active --quiet photoprism-index-all.service && exit 1 || exit 0'
ExecStart=docker exec photoprism_photoprism_1 /photoprism/bin/photoprism cleanup
ExecStartPost=docker exec photoprism_photoprism_1 /photoprism/bin/photoprism purge
16:19 ♒♒♒   ☺ 😄    
```

**Comment by lastzero** [comment]
@rickysarraf @vadi2 I suspect there could have been an issue with the database connection as the `photoprism` service was not running but `mariadb` was.

- Since the password is stored in the database, it affects authentication. Even when connecting works, there can be database (user and storage folder) permission issues.
- When the database was already initialized in an earlier attempt, PhotoPrism also won't change the existing password in there - so changing the environment variable doesn't have any effect.
- To test this, simply observe the server logs (as described in our docs or the Docker manual) and make sure you start with a clean database, e.g. by manually deleting the files in the database storage folder while containers are down or stopped.
- The API itself most likely is not broken because we actively use it in this very moment just like thousands of other users. Also we have unit and acceptance tests for this. At best API issues can be cause by a bad request encoding. In that case Error 400 might be justified. Using a common Web browser usually fixes this. Sometimes people also have plugins or desktop firewalls installed that do strange things, for example remove the password from the request because they think it's getting stolen. The list of possibilities is endless.

We've spent the last few days improving our docs in reaction to this:

https://docs.photoprism.org/getting-started/docker-compose/

For example, we've added troubleshooting tips and added more CLI command examples.
